README.md

```diff
@@ -28,3 +28,20 @@ In diagrammatic form this looks like the image below:
 ![omSupply Architecture drawio](./architecture.drawio.svg)
 
 There are also several ancilliary applications as noted, which are available to be run (cli applications) or hosted (storybook) separately.
+
+### Builds
+
+We have some automations to build windows installers based on git tags. To create a build:
+
+1. Update the package version in `client/package.json` appropriately and commit it. [v1.0.4 example](https://github.com/openmsupply/open-msupply/blob/18b193ae0ecd16a5c48190a2a346cb459eeed30d/client/package.json#L3)
+2. Create a tag on your commit stating the version/build you're doing, but add `v` to the start. [v1.0.4 example](https://github.com/openmsupply/open-msupply/tree/v1.0.4)
+3. Our [Jenkins server]([url](https://jenkins.msupply.org:8443/)) will pickup this tag and start the build process against the repo for the tagged commit
+4. A TMF staff member can grab the executables generated and share them on the [releases](https://github.com/openmsupply/open-msupply/releases) page.
+
```

Comment:
```suggestion
#### Android
The current android build requires a linux host and is a manual process. Refer to the [android build readme](https://github.com/openmsupply/open-msupply/tree/develop/client#android-app) for further details. 
```

we have a linux host: build.msupply.org for this purpose - since this is for internal use, I don't think we should add that to the readme.

Reply:
Note, the new branch Andrei is working on doesn't need a linux host anymore, does it?